src/backend_stub/logic.js

```diff
@@ -208,13 +208,23 @@ new Writer(_dataStreams['memory'], 1000, function (callbackFn) {
 var TEST_DATA_STREAMS_COUNT = 10;
 
 (function () {
-	for (var ic = TEST_DATA_STREAMS_COUNT, i = 0; i < ic; ++i) {
-		_dataStreams['data' + i] = new DataStream();
-		new Writer(_dataStreams['data' + i], (i + 1) * 1000, function (callbackFn) {
+	function makeWriter(i, dataStream) {
+		var seed = Math.random();
+		var index = 0;
+		new Writer(dataStream, (i + 1) * 200, function (callbackFn) {
 			var now = Date.now();
-			callbackFn(now, Math.abs(0.6 * Math.cos(now) + 0.4 * Math.random()));
+			callbackFn(now, Math.cos(index * (2 * Math.PI) / 10 - seed * 100));
+			++index;
```

Comment:
Also, why `10`? W/o reading the rest of the code, would `TEST_DATA_STREAMS_COUNT` be more appropriate?